cmd/chronicle/cli/options/create.go

```diff
@@ -0,0 +1,71 @@
+package options
+
+import (
+	"fmt"
+
+	"github.com/anchore/chronicle/chronicle/release/format"
+	"github.com/anchore/fangs"
+)
+
+type Create struct {
```

Comment:
This is really a command and not an option, so it should be in the `commands` package.